src/main/webapp/about.html

```diff
@@ -24,6 +26,52 @@
           <i class="fas fa-bars"></i>
       </div>
     </header>
-    <h1 class="title">about page to be added</h1>
+    <div id="main">
+      <h1 class="title">why noms?</h1>
+      <div class="section">
+        <div class="heading">
+          <img class="heading-img" src="./assets/svg/environment.svg" alt="Heart with seedling">
+          <h3 class="heading-text">
+            for the environment
+          </h3>
+        </div>
+        <p class="section-text">
+          When catered campus events end, the leftover food needs to be consumed quickly or risk being thrown away. 
+          This kind of potential food waste is, environmentally speaking, the worst kind there is: food that has 
+          completed the entire cycle of growing, transportation, storage, and preparation—just to end up in the bin.
+          <strong>noms</strong> helps divert some of the 150,000 tons of food destined for the landfill every day into
```

Comment:
I think it would be super cute to make the **noms** be a different color in our palette colors!

Reply:
Ok! I'll try out the change :)

Reply:
I tried it out and I ran into a couple issues:
- Our palette colors aren't vibrant/dark enough to be used at that font size and still be AAA accessible
- If I try to either up the vibrancy or darken the color so that the color is AAA accessible, it looks weirdly out of place with the rest of the color palette (might just be me!), or it ends up being dark enough that it might as well be black
![image](https://user-images.githubusercontent.com/35010111/90420461-d7cf5380-e06c-11ea-8be4-de478c84f37e.png)
![image](https://user-images.githubusercontent.com/35010111/90420731-34327300-e06d-11ea-8710-a292c363891b.png)

I think for those reasons + we still have some of our palette visible in the subtitles of the page + lots of colors from the icons, I'm going to leave the bolded parts black :)